charts/redis-ha/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 4.22.2
+version: 4.22.3
 appVersion: 7.0.4
```

Comment:
@DandyDeveloper should I have bumped appVersion to 7.0.5 and version to 4.23.0 ?